packages/apputils-extension/src/settingconnector.ts

```diff
@@ -59,8 +59,8 @@ export class SettingConnector extends DataConnector<
     }
 
     return {
-      ids: ids.filter(id => !isDeferred(id) && !isDisabled(id)),
-      values: values.filter(({ id }) => !isDeferred(id) && !isDisabled(id))
```

Comment:
Just to be sure, this will load the settings even for deferred extensions avoiding multiple calls (in particular if they contain definition to be gathered by a transform operation?)

Reply:
Deferred extension settings should be loaded in exactly the same way as non-disabled extension settings.
I think the settings for deferred extensions were not loaded at all before.